### PR TITLE
Test DurationFormat

### PR DESCRIPTION
**Pull request by romulocintra**
Add initial tests to Intl.DurationFormat proposal #3305 

Proposal: https://github.com/tc39/proposal-intl-duration-format

### CODE REVIEW

**Comment by rwaldron** [comment]
@romulocintra is this intentionally still in draft mode?

**Comment by romulocintra** [comment]
> @romulocintra is this intentionally still in draft mode?

Yes, it was mostly because there are plenty of missing tests, but might do it progressively instead of a big batch of tests. I think will be best to review

**Comment by rwaldron** [comment]
> > @romulocintra is this intentionally still in draft mode?
> 
> Yes, it was mostly because there are plenty of missing tests, but might do it progressively instead of a big batch of tests. I think will be best to ease the review, It might be worth creating a TODO list in the #3305 🤔🤔

Yeah, small PRs definitely move quicker. Track however works best for you

File: test/intl402/DurationFormat/instance/length.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+features: [Intl.DurationFormat]
+includes: [propertyHelper.js]
+---*/
+
+assert.sameValue(Intl.DurationFormat.length, 0);
+
+verifyNotEnumerable(Intl.DurationFormat, "length");
+verifyNotWritable(Intl.DurationFormat, "length");
+verifyConfigurable(Intl.DurationFormat, "length");
```
Comment (rwaldron) [inline test/intl402/DurationFormat/instance/length.js:32]: In the future, you could also write: 

```js
verifyProperty(Intl.DurationFormat, 'length', {
  enumerable: false,
  writable: false,
  configurable: true
});
```

**Review by rwaldron — APPROVED** [review]

File: test/intl402/DurationFormat/instance/length.js (hunk 1 above)
Comment (romulocintra) [reply test/intl402/DurationFormat/instance/length.js:32]: Thanks will do for sure

File: test/intl402/DurationFormat/prototype/format/length.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    value is equal to the largest number of named arguments shown in the
+    subclause headings for the function description. Optional parameters
+    (which are indicated with brackets: [ ]) or rest parameters (which
+    are shown using the form «...name») are not included in the default
+    argument count.
+    Unless otherwise specified, the length property of a built-in function
+    object has the attributes { [[Writable]]: false, [[Enumerable]]: false,
+    [[Configurable]]: true }.
+
+includes: [propertyHelper.js]
```
Comment (iamstolis) [inline test/intl402/DurationFormat/prototype/format/length.js:24]: This test is missing `features: [Intl.DurationFormat]`.

File: test/intl402/DurationFormat/prototype/formatToParts/length.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    value is equal to the largest number of named arguments shown in the
+    subclause headings for the function description. Optional parameters
+    (which are indicated with brackets: [ ]) or rest parameters (which
+    are shown using the form «...name») are not included in the default
+    argument count.
+    Unless otherwise specified, the length property of a built-in function
+    object has the attributes { [[Writable]]: false, [[Enumerable]]: false,
+    [[Configurable]]: true }.
+
+includes: [propertyHelper.js]
```
Comment (iamstolis) [inline test/intl402/DurationFormat/prototype/formatToParts/length.js:24]: This test is missing `features: [Intl.DurationFormat]`.

File: test/intl402/DurationFormat/prototype/resolvedOptions/length.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    value is equal to the largest number of named arguments shown in the
+    subclause headings for the function description. Optional parameters
+    (which are indicated with brackets: [ ]) or rest parameters (which
+    are shown using the form «...name») are not included in the default
+    argument count.
+    Unless otherwise specified, the length property of a built-in function
+    object has the attributes { [[Writable]]: false, [[Enumerable]]: false,
+    [[Configurable]]: true }.
+
+includes: [propertyHelper.js]
```
Comment (iamstolis) [inline test/intl402/DurationFormat/prototype/resolvedOptions/length.js:24]: This test is missing `features: [Intl.DurationFormat]`.

**Comment by Bert66cool** [comment]
Thank you
Sent from Yahoo Mail on Android 
 
  On Tue, Jan 25, 2022 at 1:11 AM, Jan ***@***.***> wrote:   



@iamstolis commented on this pull request.

In test/intl402/DurationFormat/prototype/resolvedOptions/length.js:
> +  Intl.DurationFormat.prototype.resolvedOptions ()
+
+  17 ECMAScript Standard Built-in Objects:
+
+    Every built-in function object, including constructors, has a length
+    property whose value is an integer. Unless otherwise specified, this
+    value is equal to the largest number of named arguments shown in the
+    subclause headings for the function description. Optional parameters
+    (which are indicated with brackets: [ ]) or rest parameters (which
+    are shown using the form «...name») are not included in the default
+    argument count.
+    Unless otherwise specified, the length property of a built-in function
+    object has the attributes { [[Writable]]: false, [[Enumerable]]: false,
+    [[Configurable]]: true }.
+
+includes: [propertyHelper.js]

This test is missing features: [Intl.DurationFormat].

File: test/intl402/DurationFormat/prototype/format/length.js (hunk 2 above)
Comment (romulocintra) [reply test/intl402/DurationFormat/prototype/format/length.js:24]: This PR fixes that https://github.com/tc39/test262/pull/3385 , thanks for the comment

Comment (iamstolis) [reply test/intl402/DurationFormat/prototype/format/length.js:24]: Thanks a lot for a very fast response and fix.

**Pull request merged**